index.html

```diff
@@ -0,0 +1,723 @@
+              alt="card image"
+              width="200"
+              height="200"
+            />
+            <div class="card-text">
+              <h4>
+                Make sense of the day’s news and ideas. David Leonhardt and
+                Times journalists guide you through what’s happening — and why
+                it matters.
+              </h4>
+            </div>
+          </div>
+          <!--THREE-->
+          <div class="image-card">
+            <img
+              src="https://github.com/Stephanie041996/Newyork/blob/pagefeatures/images/merlin_144381195_afc81c81-6721-4fe7-beab-1c08a72f5c71-threeByTwoSmallAt2X.jpg"
+              alt="card image"
+              width="200"
+              height="200"
+            />
+            <div class="card-text">
+              <h4>
+                Make sense of the day’s news and ideas. David Leonhardt and
+                Times journalists guide you through what’s happening — and why
+                it matters.
+              </h4>
+            </div>
+          </div>
+          <!--Third ROW -->
+          <!--ONE-->
+
+          <div class="image-card">
+            <img
+              src="https://github.com/Stephanie041996/Newyork/blob/pagefeatures/images/15kaliningrad1-threeByTwoSmallAt2X-v30.jpg"
+              alt="card image"
+              width="200"
+              height="200"
+            />
+            <div class="card-text">
+              <h4>
+                Make sense of the day’s news and ideas. David Leonhardt and
+                Times journalists guide you through what’s happening — and why
+                it matters.
+              </h4>
+            </div>
+          </div>
+
+          <!--TWO-->
+          <div class="image-card">
+            <img
+              src="https://github.com/Stephanie041996/Newyork/blob/pagefeatures/images/00SCI-VENUS1-threeByTwoSmallAt2X.jpg"
+              alt="card image"
+              width="200"
+              height="200"
+            />
+            <div class="card-text">
+              <h4>
+                Make sense of the day’s news and ideas. David Leonhardt and
+                Times journalists guide you through what’s happening — and why
+                it matters.
+              </h4>
+            </div>
+          </div>
+          <!--THREE-->
+          <div class="image-card">
+            <img
+              src="https://github.com/Stephanie041996/Newyork/blob/pagefeatures/images/merlin_144381195_afc81c81-6721-4fe7-beab-1c08a72f5c71-threeByTwoSmallAt2X.jpg"
+              alt="card image"
+              width="200"
+              height="200"
+            />
+            <div class="card-text">
+              <h4>
+                Make sense of the day’s news and ideas. David Leonhardt and
+                Times journalists guide you through what’s happening — and why
+                it matters.
+              </h4>
+            </div>
+          </div>
+        </div>
+      </section>
```

Comment:
## Implement the footer section
- [ ] The footer section has not been implemented. Let's implement it.
- [ ] Also the page need to have the fixed footer section in place as we.
- ![bottom-footer](https://user-images.githubusercontent.com/22345961/108400451-3fd54100-722c-11eb-8c05-31ea01c0d458.png)